tests/taskcluster/tasks/eslint.yml

```diff
@@ -0,0 +1,31 @@
+task:
+  metadata:
+    name: Gaia ESLint
+    description: JS linter
+
+  workerType: '{{workerType}}'
+  provisionerId: '{{provisionerId}}'
+  scopes:
+    # Source caches contains various sources (including a tarball of gaia source
+    # tree)
+    - docker-worker:cache:gaia-misc-caches
+    - docker-worker:image:quay.io/mozilla/gaia-taskenv:*
+
+  payload:
+    image: quay.io/mozilla/gaia-taskenv:0.8.9
```

Comment:
Do not specify the image to use, it's taken care of for you. This is infact an outdated image that you're trying to use.